pwndbg/aglib/file.py

```diff
@@ -43,6 +45,29 @@ def get_proc_exe_file() -> str:
     return get_file(pwndbg.aglib.proc.exe, try_local_path=True)
 
 
+@pwndbg.lib.cache.cache_until("start")
+def can_download_remote_file() -> bool:
+    if not pwndbg.aglib.remote.is_remote():
+        return False
+    elif pwndbg.aglib.qemu.is_qemu_kernel():
+        return False
+
+    # Some[1] gdb servers don't implement vFile packets.
+    # [1] - qemu-user <8.1
+    # [1] - Rosetta2
+    # [1] - maybe embedded probe like: `Black Magic Probe V2.3`
+    # WTF: There is no indication in `qSupported` when `vFile` packets are supported
```

Comment:
Does qemu-user>=8.1 do indicate this?

Is this maybe a bug in Rosetta2 and embedded probes?

Reply:
`qemu-user>=8.1` sends only `qXfer:exec-file:read` but what this mean at all? 
https://sourceware.org/legacy-ml/gdb-testers/2015-q2/msg02348.html